Compile error with JSON option

when compiling with the JSON option
```
⟩ cmake -DCMAKE_BUILD_TYPE=Release -DCLARABEL_FEATURE_SDP=sdp-openblas -DCLARABEL_FEATURE_SERDE=true ..                                                                                

⟩ make
   Compiling clarabel_c v0.1.0 (/home/matbesancon/Documents/crptests/Clarabel.cpp/rust_wrapper)
error[E0425]: cannot find function `get_solver_settings_from_c` in module `utils`
   --> src/solver/implementations/default/solver.rs:317:31
    |
317 |         let settings = utils::get_solver_settings_from_c::<T>(settings_struct);
    |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `utils`

For more information about this error, try `rustc --explain E0425`.
error: could not compile `clarabel_c` (lib) due to 1 previous error
make[2]: *** [rust_wrapper/CMakeFiles/libclarabel_c.dir/build.make:70: rust_wrapper/CMakeFiles/libclarabel_c] Error 101
make[1]: *** [CMakeFiles/Makefile2:201: rust_wrapper/CMakeFiles/libclarabel_c.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

Additional info:
```
rustc 1.87.0
```

--- a/rust_wrapper/src/solver/implementations/default/solver.rs
+++ b/rust_wrapper/src/solver/implementations/default/solver.rs
@@ -313,8 +313,7 @@ where
     let solver = if settings.is_null() {
         lib::DefaultSolver::<T>::load_from_file(&mut file, None)
     } else {
-        let settings_struct = &*(settings);
-        let settings = utils::get_solver_settings_from_c::<T>(settings_struct);
+        let settings = (*settings).clone().into();
         lib::DefaultSolver::<T>::load_from_file(&mut file, Some(settings))
     };
     Box::into_raw(Box::new(solver)) as *mut c_void

